jsx/Tabs.js

```diff
@@ -173,10 +171,7 @@ TabPane.propTypes = {
   activeTab: React.PropTypes.string
 };
 
-window.Tabs = Tabs;
-window.TabPane = TabPane;
-
-export default {
+export {
```

Comment:
@jacobpenny I didn't work with `default`. (I guess export default has to be named or something)